instrumentation/apache-httpclient/apache-httpclient-5.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/apachehttpclient/v5_0/AbstractApacheHttpClientTest.java

```diff
@@ -42,7 +44,8 @@ protected Set<AttributeKey<?>> getHttpAttributes(URI endpoint) {
     attributes.add(SemanticAttributes.HTTP_URL);
     attributes.add(SemanticAttributes.HTTP_METHOD);
     if (endpoint.toString().contains("/success")) {
-      attributes.add(SemanticAttributes.HTTP_FLAVOR);
+      attributes.add(stringKey("net.protocol.name"));
+      attributes.add(stringKey("net.protocol.version"));
     }
```

Comment:
just checking, is this currently limited to `/success`

Reply:
Nope, it actually works in several other scenarios; fixed.